lang/cs/Org.Apache.REEF.Wake/StreamingRemote/lang/cs/Org.Apache.REEF.Wake/Remote/ICodec.cs

```diff
@@ -0,0 +1,34 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace Org.Apache.REEF.Wake.Remote
+{
+    public interface ICodec
```

Comment:
Its the same one..I did not create my own...the only file u need to look at
are the ones mentioned in the pull request..others are the same

On Thu, Apr 16, 2015 at 4:16 PM, Markus Weimer <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.Wake/StreamingRemote/lang/cs/Org.Apache.REEF.Wake/Remote/ICodec.cs
> <https://github.com/apache/incubator-reef/pull/149#discussion_r28558668>:
>
> > + * "License"); you may not use this file except in compliance
> > + * with the License.  You may obtain a copy of the License at
> > + *
> > + *   http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing,
> > + * software distributed under the License is distributed on an
> > + * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > + * KIND, either express or implied.  See the License for the
> > + * specific language governing permissions and limitations
> > + * under the License.
> > + */
> > +
> > +namespace Org.Apache.REEF.Wake.Remote
> > +{
> > +    public interface ICodec
>
> Don't we have a ICodec already?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/149/files#r28558668>.
>